cal/cal/models.py

```diff
@@ -583,20 +584,38 @@ def query(self, calendar_ids=None, start=None, end=None):
 
         return events_qs
 
-    def get_hours_per_week(self, calendar_ids=None, start=None, end=None):
+    def get_hours_per_week(self, timezone, calendar_ids=None, start=None, end=None):
```

Comment:
There's probably a way to get the string representation of a timezone using pytz